### ISSUE TITLE
Yaml file inconsistent casing

### ISSUE THREAD

**Issue by srkirkland** [ucdavis/hippo#149]

**Comment by OmenWild** [comment]
Specifically the `Account` key:

```sponsor:
    accountname: Scott's Account
    name: Scott Kirkland
    email: srkirkland@ucdavis.edu
    kerb: postit
    iam: 1000029584
    mothra: 00183873

Account:
    name: Jason Sylvestre2
    email: jcsylvestre@ucdavis.edu
    kerb: xxxx
    iam: 1000091412
    mothra: 00842755```

**Comment by srkirkland** [comment]
fixed by #150
